AICP: {
  Registration: {
    type: "Pointmatcher", # GICP, Kaess
    sensorRange: 10.0,         # in meters
    sensorAngularView: 360.0,  # in degrees
    loadPosesFrom: "file", # file (ground truth poses), pcd
    initialTransform: "random", # x[m], y[m], yaw[deg]
                                     # - if "random" initialTransform will be
                                     # sampled from Gaussian distributed perturbations
                                     # with 0.1 m and 10 deg variance magnitude
                                     # - if "" no initial tranform will be applied
    saveCorrectedPose: true, # idA, idB, x, y, z, w, x, y, z
                             # (valid if loadPosesFrom)
    saveInitializedReadingCloud: true,
    saveRegisteredReadingCloud: true,
    enableLcmVisualization: true,

    Pointmatcher: {
      configFileName: "icp_autotuned_default.yaml",
      printOutputStatistics: false,
    }
  },
  Overlap: {
    type: "OctreeBased",
 #   loadPosesFromFile: "pose_scanner_leica_affine_apartment.txt", # Ground truth poses

    OctreeBased: {
      octomapResolution: 0.2,
    }
  },
  Classifier: {
    type: "SVM",

    SVM: {
      threshold: 0.50,
      trainingFile: "/home/edbot/rpg-navigation/software/aicp/data/labels/training_all_labelled_1Sep.txt",
      testingFile: "/home/edbot/rpg-navigation/software/aicp/data/labels/testing_labelled_27Aug.txt",
      saveFile: "/home/edbot/rpg-navigation/software/aicp/data/classification/svm_1000training_thresh50_cross_validation.xml",
      saveProbs: "/home/edbot/rpg-navigation/software/aicp/data/classification/probs.txt",
      modelLocation: "/home/edbot/rpg-navigation/software/aicp/data/models/svm.xml",
    },
  },
  Experiments: {
    type: "Compare", # "",
                     # "Validation" saves: idA, idB, alignability, degeneracy,
                     # "Compare" saves: idA, idB, fov_overlap, octree_overlap,
                     #                  alignability, align_risk, degeneracy, ICN
  }
}
